performances/src/trackNetwork.ts

```diff
@@ -1,36 +1,53 @@
 import type { HTTPRequest, Page } from 'puppeteer'
 
+const REQUEST_TIMEOUT = 5000 // 5 seconds
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it may be more useful to explain why 5s